# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gpe-shield/uclibc.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- gpe-shield-0.31/interface.c.vanilla	2007-08-14 10:33:15.000000000 +0200
+++ gpe-shield-0.31/interface.c	2007-08-14 10:34:37.000000000 +0200
@@ -18,7 +18,9 @@
 #include <unistd.h>
 #include <dirent.h>
 #include <fcntl.h>
+#ifndef __UCLIBC__
 #include <stropts.h>
+#endif
 #include <poll.h>
 #include <sys/types.h>
 #include <sys/socket.h>
@@ -37,7 +39,6 @@
 #include <unistd.h>
 #include <dirent.h>
 #include <fcntl.h>
-#include <stropts.h>
 #include <poll.h>
 #include <sys/types.h>
 #include <sys/socket.h>
